package.json

```diff
@@ -16,7 +16,7 @@
     "build:prod": " ./node_modules/.bin/webpack --config webpack.config.js --env prod",
     "clean": "rm -rf dist/* db/development.sqlite && npm run prepcerts",
     "test": "jest",
-    "start": "node ./server/index.js",
+    "start": "./run serve",
```

Comment:
I think I merged this in as part of your previous change (conditionally create directories). Let's work through this in person the day before the workshop and figure out exactly what we want here.